### ISSUE TITLE
1.18.x - Biomechange not work on new highlimits 

### ISSUE THREAD

**Issue by Chtholly** [BentoBoxWorld/Biomes#97]
- Changing a biome of the island/chunk only affect from high 0-320
- in minus high biome dont change
(unload chunks / leaving server already done)

### BentoBox version
Bentobox: 1.19.0
AOneBlock: 1.9.0
Biomes: 1.14.0

**Comment by Chtholly** [comment]
- Testing with Biomes-1.14.1-SNAPSHOT-b214.jar same result

**Comment by BONNe** [comment]
It will be implemented in version 2.0.0

Currently, it is blocked by https://github.com/BentoBoxWorld/BentoBox/pull/1919
